src/Controls/src/Core/HandlerImpl/ContentPage.Impl.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Maui.Controls
+{
+	// TODO: We don't currently have any concept of a page in Maui
+	// so this just treats it as a layout for now
+	public partial class ContentPage : Microsoft.Maui.ILayout
```

Comment:
Can we do a IPage instead since we have that? or better in a new PR

IPage : ILayout

?

Reply:
Initially I had done that...   In the end I don't think `IPage : ILayout` so I'm just going for a really light touch approach here.

Once someone implements a PageHandler then I think all this will shake out in a more accurate way